sys/i386/linux/syscalls.master

```diff
@@ -1554,7 +1554,7 @@
 281	AUE_NULL	STD {
 		int linux_mq_notify(
 		    l_mqd_t mqd,
-		    const struct l_timespec *abs_timeout
+		    const struct sigevent *sevp
```

Comment:
Why sigevent and not l_sigevent?

Reply:
> Why sigevent and not l_sigevent?

Goog catch.  Btw, we have sigevent in sys/amd64/linux/syscalls.master for linux_timer_create.